<launch>
  <arg name="main" default="true" />
  <arg name="can_name" default="georgia" />

  <node if="$(arg main)" pkg="fridge_demo_bt" type="fridge_demo" name="fridge_demo_main"
        args="--arg can_name $(arg can_name)"  output="screen" />

  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
    <arg name="unregister_timeout" value="100000"/>
  </include>

  <node pkg="roseus" type="roseus" name="fridge_demo_action_server"
        args="$(find fridge_demo_bt)/euslisp/fridge-demo-action-server.l"
        output="screen"/>

</launch>
